source/index-warning.txt

```diff
@@ -6,10 +6,103 @@ Missing Index Warning
 
 .. default-domain:: mongodb
 
+.. meta:: 
+   :description: The MongoDB for IntelliJ Plugin examines Java code for missing indexes.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44966.
+Definition
+----------
+
+The {+intellij-full+} examines if application queries use indexes. If a
+query doesn't use an index, the plugin displays a warning for that
+query.
+
+Before you add an index, consider if:
+
+- the query runs often enough to justify reducing the write performance
+  for faster reads.
+- you can change the query to use an existing index.
+
+To resolve the warning, consider creating an index for the query. You
+can also disable index warnings.
+
+For more information about indexes, see :ref:`manual-create-an-index`.
+
+Examples
+--------
+
+In the following example Java code snippet, the ``awards`` document
+field is used in a query, but the field isn't indexed in the database:
+
+.. code-block:: java
+   :copyable: false
+   :emphasize-lines: 2
+
+   client.getDatabase( "sample_database" ).getCollection( "movies" ).find(
+      Filters.ne( "awards", "Comedy" )
+   )
+
+The plugin shows this warning:
+
+.. code-block:: none
+   :copyable: false
+
+   This query will run without an index. If you plan on using this
+   query heavily in your application, you should create an index that
+   covers this query.
+   
+   Implement an Index
+
+Create an Index
+~~~~~~~~~~~~~~~
+
+To create an index for the query, click the :guilabel:`Implement an
+Index` link displayed with the warning in the plugin. The plugin then
+displays the :guilabel:`Database Explorer Playgrounds` screen with
+template code that shows potential fields to index in a comment. For
```

Comment:
[suggestion] This sentence that starts with "The plugin then displays..." seems like a run-on sentence. Could we maybe split this up? 

Maybe something along the lines of: 
```
The plugin then displays the :guilabel:`Database Explorer Playgrounds` screen with
template code for creating an index. The template code also includes a comment 
that shows potential fields to index. For example...
```

Reply:
Done